packages/transactions/src/PaymentV2.ts

```diff
@@ -20,11 +20,12 @@ interface PaymentOptions {
   signature?: Uint8Array
 }
 
-interface Payment {
+export interface Payment {
   payee: Addressable
   amount: number
   memo?: Base64Memo
   tokenType?: string
+  max?: boolean | null
```

Comment:
Do we need the `| null` with the `?`

Reply:
That's how the proto is defined, but I suppose it would be better to keep it more in line with how this package handles that.